packages/cli-kit/src/private/node/ui/components/TokenizedText.tsx

```diff
@@ -56,6 +61,7 @@ export type Token =
   | FilePathToken
   | ListToken
   | BoldToken
+  | ColoredToken
```

Comment:
pinging @matteodepalo because you did all the UI stuff, just so that you are aware of this change

Reply:
ah! you were faster :P